app/Providers/Tools/TelescopeServiceProvider.php

```
@@ -0,0 +1,67 @@
+
+class TelescopeServiceProvider extends TelescopeApplicationServiceProvider
+{
+    /**
+     * Register any application services.
+     */
+    public function register(): void
+    {
+        // Telescope::night();
+        $this->hideSensitiveRequestDetails();
+
+        Telescope::filter(fn (IncomingEntry $entry) => true);
```

PHPMD (2.15.0)

Avoid unused parameters such as '$entry'. (undefined)

(UnusedFormalParameter)